x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -75,12 +75,10 @@ export async function getPackages(
   const packageSavedObjects = await getPackageSavedObjects(savedObjectsClient);
 
   const uploadedPackagesNotInRegistry = packageSavedObjects.saved_objects
-    .filter(
-      (pkg) =>
-        pkg.attributes.install_source === 'upload' &&
-        !registryItems.some((item) => item.name === pkg.id)
-    )
-    .map((pkg) => createInstallableFrom({ ...pkg.attributes, title: nameAsTitle(pkg.id) }, pkg));
+    .filter((pkg) => !registryItems.some((item) => item.name === pkg.id))
+    .map((pkg) =>
+      createInstallableFrom({ ...pkg.attributes, title: nameAsTitle(pkg.id), id: pkg.id }, pkg)
```

Comment:
Found a better solution, by adding an id, there is no more duplication in the cards.
By not filtering only on `install_source:upload`, this solves for seeing installed beta integrations even when beta integrations switch is off.

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/90178898/218432592-fa8cdb99-eb40-4063-aab8-c717d0a76bbf.png">